charts/bitcoind/templates/statefulset.yaml

```diff
@@ -156,6 +197,11 @@ spec:
         {{- else }}
           emptyDir: {}
         {{- end -}}
+        {{- if .Values.descriptorImport.enabled }}
```

Comment:
Don't think we need a flag.
Perhaps we can pass in:
`.Values.descriptor.{secretName, secretKey}` and if both are present - then that is the indication that we want to load the descriptor.

Reply:
done, checking for a the `secretName` and `secretKey` now